obelix/__init__.py

```diff
@@ -279,24 +298,120 @@ def read_data(self, data_path, no_cifs=False):
 class ShonAndMin(Dataset):
     '''
     Shon and Min dataset class.
-    
+
     Attributes:
-        dataframe (pd.DataFrame): DataFrame containing the dataset.
+        dataframe (pd.DataFrame): Cleaned DataFrame containing the dataset.
     '''
 
-    def __init__(self, data_path="/home/leah/leah---OBELiX/data/Shon_and_Min.xlsx"):
+    def __init__(self, data_path="./SM_rawdata"):
         '''
-        Loads the Shon and Min dataset from an Excel file.
+        Loads and cleans the Shon and Min dataset from Sheet2.
         '''
         self.data_path = Path(data_path)
-        df = self.read_data(self.data_path)
-        super().__init__(df)
-        self.dataframe = df
-    
-    def read_data(self, data_path):
-        '''Reads the Shon and Min dataset.'''
-        data = pd.read_excel(data_path)
+        self.data_path.mkdir(parents=True, exist_ok=True)
+
+        self.source_file = Path("/home/leah/leah---OBELiX/data/ao3c01424_si_001.xlsx")
```

Comment:
Replace this by the url. This would only work on your computer anyway since it is an absolute path

Reply:
Replace this by a URL to our repo

Reply:
Make it a link to your Github for now and we will change it once we merge into main here